### PR TITLE
Give the readme a little love #163

### PR DESCRIPTION
**Pull request by markwilliamfirth**
Give the readme a little love #163

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
-# dwyl hq
-:red_circle: open kimono
+# dwyl
 
-# Why?
+DWYL LTD is incorporated and registered in England and Wales with company number 09525434 and registered office is at DWYL LTD, 104 Mary Datchelor Close, London, SE5 7DY, United Kingdom.
```
Comment (iteles) [inline README.md]: I think this needs a *tiny* bit of expansion, I always talk about first and foremost being an open source community and that is the friendliness I'd like to convey, see: https://github.com/dwyl/start-here/issues/117

I think that that and the mission is way *more important* than the registered company for now, particularly with regards to our github audience (vs our website audience which is more client-focused) and should be the first thing people read :blush:

Comment (markwilliamfirth) [inline README.md]: I thought it would be handy considering it's the ops and admin repo

Comment (iteles) [inline README.md]: I agree it's important info, particularly the registered company no. and address, I'm just requesting we move the info around to reflect what we want our audience to see first - most people who get here don't know dwyl as anything more than an awesome online resource!

**Comment by iteles** [comment]
I feel like I just wore you down here 😆

**Comment by markwilliamfirth** [comment]
Is it possible for reviewers to edit the file directly rather than suggest changes? I wouldn't be offended if you wanted to make a change like that when reviewing! I misread your first message and thought you didn't want it rather than just moving it down

**Comment by iteles** [comment]
@markwilliamfirth It is, but then it would still require a review from you.

For small changes like this and typos it's probably the better way 👍

**Comment by markwilliamfirth** [comment]
The former might be more beneficial - probably would have been quicker (in terms of our combined time) rather than explaining etc

with me reviewing i'd see and understand what you'd want changing

**Pull request merged**